src/app/modules/products/pages/product/product.page.html

```diff
@@ -9,10 +9,13 @@
             </ng-template>
         </ng-container>
     </app-grid-list>
+    <ng-container *ngIf="products$ | async as products">
+        <app-filter [filterOptions]="getFilterableProperties(products)" (selectedOptions)="onSelectionChange($event)"></app-filter>
```

Comment:
I wasn't sure how you wanted to filter styled, so I've just left it as it comes out the box at the moment. Let me know if you want it styled in a specific way.